analyzers/Merger.py

```diff
@@ -0,0 +1,19 @@
+from heppy.framework.analyzer import Analyzer
+import copy
+
+
+
+class Merger(Analyzer):
```

Comment:
Hi Alice, 

Thanks for these modifications!
Could you please add doctest to this class as for other analyzers?